src/main/java/godaddy/registry/jtoolkit2/se/ContactInfoResponse.java

```diff
@@ -183,7 +182,7 @@ public void fromXML(XMLDocument xmlDoc) {
             String iCC = xmlDoc.getNodeValue(CON_PINFO_INT_CC_EXPR);
 
             intPostalInfo = new IntPostalInfo(iName, iOrg, iStreet, iCity,
-                    iSP, iPC, iCC);
+                iSP, iPC, iCC);
```

Comment:
Could you please undo this change? It's irrelevant to the fix.

Reply:
Done